jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/scrollspy/ScrollspyInNavbarTests.java

```diff
@@ -0,0 +1,195 @@
+                    .attr("href", is(link));
+        } else {
+            navbarWithDropdown.dropdownMenu.expand();
+            navbarWithDropdown.dropdownMenu.list().get(header).is()
+                    .core()
+                    .displayed()
+                    .enabled()
+                    .text(is(header))
+                    .value(is(header))
+                    .attr("href", is(link));
+        }
+        scrollspyInNavbar.header.get(index).is()
+                .core()
+                .displayed()
+                .enabled()
+                .text(is(header.toUpperCase()))
+                .value(is(header.toUpperCase()))
+                .attr("id", is(paragraph));
+
+        scrollspyInNavbar.mainText.get(index).is()
+                .core()
+                .displayed()
+                .enabled()
+                .text(is(mainText))
+                .value(is(mainText));
+    }
+
+    @Test
+    public void isValidationTests() {
+        navbarWithDropdown.navItemLink.get(3).is().text(dropdown);
+        navbarWithDropdown.navItemLink.get(3).is().value(dropdown);
+        navbarWithDropdown.navItemLink.is().size(3);
+        navbarWithDropdown.navGroup.is().size(3);
+
+        navbarWithDropdown.dropdownMenu.expand();
+        navbarWithDropdown.dropdownMenu.is().size(3);
+
+        navbarWithDropdown.find(By.className("dropdown-divider")).is()
+                .core()
+                .displayed()
+                .enabled()
+                .attr("role", "separator");
+
+        scrollspyInNavbar.mainText.is().size(5);
+        scrollspyInNavbar.header.is().size(5);
+
+        navbarWithDropdown.navbarLink.is()
+                .core()
+                .displayed()
+                .enabled()
+                .attr("href", is(navbarLink))
+                .value(is(navbarText))
+                .text(is(navbarText))
+                .cssClass("navbar-brand");
+
+        for (UIElement element : navbarWithDropdown.navGroup.list()) {
+            element.is()
+                    .core()
+                    .displayed()
+                    .enabled()
+                    .hasClass("nav-item");
+        }
+
+        scrollspyInNavbar.is()
+                .core()
+                .displayed()
+                .enabled()
+                .attr("data-spy", is("scroll"))
+                .attr("data-target", is("#navbar-example2"))
+                .attr("data-offset", is("0"))
+                .cssClass("scrollspy-example");
+    }
+
+    @Test(dataProvider = "itemsCheck")
+    public void linkClickableFocusTests(int index, String header) {
+        navbarWithDropdown.navItemLink.get(1).click();
+        navbarWithDropdown.dropdownMenu.expand();
+
+        int y_header_start = scrollspyInNavbar.header.get(1).core().getRect().y;
+        if (index < 3)
+            navbarWithDropdown.navItemLink.get(index).click();
+        else
+            navbarWithDropdown.dropdownMenu.list().get(header).click();
```

Comment:
it's better to always use {} for **if** even when there is only one line inside:
```
        if (index < 3) {
            navbarWithDropdown.navItemLink.get(index).click();
       }
        else {
            navbarWithDropdown.dropdownMenu.list().get(header).click();
        } 
```
(as example I can provide the link: https://google.github.io/styleguide/javaguide.html#s4.1-braces )